docs/release.md

```diff
@@ -0,0 +1,52 @@
+# 📦 Release Process
+
+This document describes the steps to release a new version of `rawfile-localpv`.
+
+---
+
+## 🛠 Prerequisites
+
+- Git installed and configured
+- Access to the GitHub repository
+- Familiarity with semantic versioning (e.g., `v1.2.3`)
+- All tests must pass and documentation should be up to date
+
+---
+
+## Development Process
+
+All development process should be done targetting the `develop` branch.  \
+Pull requests must be reviewed by at least 1 codeowner and all CI tests must pass. You can find out more information on the
+testing side in the [contributor docs](./contributor.md).
+
+## Version Numbering
+
+We follow [semantic versioning](https://semver.org/) rules when releasing, ensuring consumes of the chart have reasonably defined expectations of breaking changes.
+
+This means we have a three-digit version number:
+
+- New patch versions (e.g. from 4.3.2 to 4.3.3) indicate bug fixes only. No new functionality, and forward and backward compatibility are guaranteed
+- New minor versions (e.g. from 4.3.3 to 4.4.0) indicate added functionality. Only backward compatibility is guaranteed
+- New major versions (e.g. from 4.4.0 to 5.0.0) indicate major revisions. All bets are off in terms of compatibility
+
+> [!INFORMATION]
+We've yet to release the first stable relase (we're `0.x.y`), which means minor releases fall in the purview of breaking changes
+This doesn't mean the product itself is not stable, but the API may change in a breaking way
+
+Please not that the helm chart version is devoid of a `v` prefix, though we still tag with the `v`, example: `v4.3.2`
+
+## 🚀 Steps to Release
+
+Only a codeowner may create a new release. The release should be prepared in the `develop` branch.  \
+Please read the following steps carefully:
+
+- [ ] Review and merge all pending PRs which are targetted for the release
+- [ ] Freeze `develop` branches from any changes
+- [ ] Ensure the Chart.yaml `version` and `appVersion` and set to the release version
+- [ ] Generate the helm chart docs (the pre-commit hook will do this for you)
+- [ ] Update `CHANGELOG.md` with the new version and changes
+- [ ] Raise PR to merge the changes above
+- [ ] Create a git tag with the format `v${Chart.yaml/version}` and push it!
```

Comment:
Tags are not creating automatically?

Reply:
No, tag push is what kicks off the release itself
At least that's how it is today, but we can always improve this process